.eslintrc.yml

```diff
@@ -0,0 +1,25 @@
+env:
+  browser: true
+  es2021: true
+extends:
+  - eslint:recommended
+  - plugin:@typescript-eslint/recommended
```

Comment:
Do we also want this to extend `prettier` so that it turns off syntax formatting rules that Prettier handles?